src/test/java/HomeWork/pages/FileUploadPage.java

```diff
@@ -0,0 +1,32 @@
+package HomeWork.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class FileUploadPage extends BasePage {
+    public final static By UPLOAD_BUTTON = By.cssSelector("input[id=file-submit]");
+    public final static By FILE_UPLOADED_SUCCESS = By.cssSelector("div[class='panel text-center']");
+    public final static By FILE_UPLOADED_PAGE = By.xpath("//*[@id=\"content\"]/div/h3");
+    public static String PATH_TO_LOADING_FILE = System.getProperty("user.dir") + "/src/test/resources/Test";
```

Comment:
PATH_TO_LOADING_FILE is a tests data - it shouldn't be hardcoded in Page class - move it to the test class